configs/icds-14lakh-dimagivalues-jan2020.yml

```
@@ -0,0 +1,750 @@
+
+  beneficiary_phone_numbers:
+    model: 'derived_factor'
+    dependant_field: 'person_cases_total'
+    factor: '{beneficiary_phone_number_seeding}'
+  phone_numbers:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'beneficiary_phone_numbers'
+      - 'users'  # 100% seeding
+      - 'ls_users'  # 100% seeding
+
+  pregancy_cases:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 6
+    monthly_growth: 1
+  child_cases:
+    model: 'baseline_with_growth'
+    dependant_field: 'users'
+    baseline: 100
+    monthly_growth: 1
+  ledgers_preg:
+    model: 'derived_factor'
+    dependant_field: 'pregancy_cases'
+    factor: 5
+  ledgers_child:
+    model: 'derived_factor'
+    dependant_field: 'child_cases'
+    factor: 10  # was 35 but that seems too high
+  ledgers_total:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'ledgers_preg'
+      - 'ledgers_child'
+  ledger_transactions:
+    model: 'derived_factor'
+    dependant_field: 'ledgers_total'
+    factor: 2  # 2 transactions per ledger (ever)
+  ledger_updates_preg:
+    model: 'derived_factor'
+    dependant_field: 'pregancy_cases_baseline'  # active pregnancy cases
+    factor: 0.6
+  ledger_updates_child:
+    model: 'derived_factor'
+    dependant_field: 'child_cases_baseline'  # active child cases
+    factor: 0.6
+  ledger_updates_monthly:
+    model: 'derived_sum'
+    dependant_fields:
+      - 'ledger_updates_preg'
+      - 'ledger_updates_child'
+
+  images_per_user:
+    model: 'date_range_value'
+    ranges:
+      - ['20180901', '20190301', 35]  # daily feeding
+      - ['20190401', '20190801', 60]
+      - ['20190901', '20200901', 650]  # daily feeding + THR + AG
+  images_monthly:
+    model: 'product'
+    dependant_fields:
+      - 'users'
+      - 'images_per_user'
+  images_total:
+    model: 'cumulative_limited_lifespan'
+    dependant_field: 'images_monthly'
+    lifespan: 3  # we only keep images for 3 months
+
+  # This is modelled because it accounts for significant storage in the main PG database
+  # This model could be improved or we find another way to account for this data
+  # Data taken from datadog in Jan 2019. Table size peaks at 1.2M rows (120K active users)
+  async_indicators:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 50  # inflated to be safe
+
+  # See https://docs.google.com/spreadsheets/d/1vzp5kLHBm6IXKI39LTDS2pN69uMryXuBkpl7c8sxik8/edit#gid=1536669590 for sms estimates
+  # (old https://docs.google.com/spreadsheets/d/16evR_N95TUBFlIr3D3ojasX7yN4odDOxH5mgcF6VCkQ/edit#gid=0)
+### PHASE 1 SMS
+  sms_monthly_phase1:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 10  # current usage of 10 messages per user per month (avg)
+### PHASE 2 SMS (assuming 100% phone number capture rate)
+  sms_thr:
+    model: 'derived_factor'
+    dependant_field: 'users'
+    factor: 120  # 5 messages per AWC per day x 24 days per month
```

"Number of THR per AWW per month"
```suggestion
    factor: 55  # ~2 messages per AWC per day x 24 days per month
```